.github/workflows/python.yml

```diff
@@ -42,40 +103,25 @@ jobs:
       if: runner.os == 'Windows'
       shell: cmd
       run: |
+        echo on
         echo using msvc ; >%HOMEDRIVE%%HOMEPATH%\user-config.jam
         mklink /J boost %BOOST_ROOT_1_72_0%
         cd boost
         b2 headers
+        cd ..
+        echo BOOST_ROOT=%CD%\boost>>%GITHUB_ENV%
+        echo BOOST_BUILD_PATH=%CD%\boost\tools\build>>%GITHUB_ENV%
+        echo %CD%\boost>>%GITHUB_PATH%
```

Comment:
what's this for?

Reply:
This is github actions magic to persist the environment variables and `PATH` to the next step. Otherwise the only way to do it seems to be to do `set BOOST_ROOT=...` which is only effective for that step, so we must duplicate all later commands in the step.